docs/docs/features/backlight.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Backlight
+sidebar_label: Backlight
+---
+
+Backlight is a feature used to control array of LEDs, usually placed through or under switches. Unlike [RGB Underglow](./underglow), backlight currently allows only one color per LED, also LEDs are not addressable, so you can't control individual LEDs.
```

Comment:
```suggestion
Backlight is a feature used to control array of LEDs, usually placed through or under switches. Unlike [RGB Underglow](underglow.md), backlight currently allows only one color per LED, also LEDs are not addressable, so you can't control individual LEDs.
```
We recently discovered that links break unless you use the exact path to the .md file.